source/widgets/ProjectStatus/js/configuration.js

```diff
@@ -107,56 +107,67 @@ function OctopusStatusWidgetConfiguration() {
                 return (prop("result")(data) || []).map(JSON.parse);
             };
 
-            var refreshSpacesProjectsAndEnvironmentsDropdowns = function (settings, queryUri, authToken) {
+            var refreshSpacesProjectGroupsAndEnvironmentsDropdowns = function (settings, queryUri, authToken) {
                 $spaceDropdown.empty();
                 $spaceDropdown.prop("disabled", true);
-                $projectFilterInput.empty();
+                $projectGroupDropdown.empty();
                 $projectDropdown.empty();
                 $environmentDropdown.empty();
 
                 const appendSpaceData = appendDropdownOptions($spaceDropdown, parseResult, selectId, selectName, settings ? settings.spaceId : null);
+                const appendProjectGroupData = appendDropdownOptions($projectGroupDropdown, parseResult, selectId, selectName, settings ? settings.projectGroupId : null);
                 const appendProjectData = appendDropdownOptions($projectDropdown, parseResult, selectId, selectName, settings ? settings.projectId : null);
                 const appendEnvironmentData = appendDropdownOptions($environmentDropdown, parseResult, selectId, selectName, settings ? settings.environmentId : null);
 
-                // TODO: Review - this results in undesired UX when editing an existing widget (you end up with two project names on top of each other).
-                // TODO: With all the onChange events going on here, how can we achieve this nicely using our new list endpoints? (consider projects that do _not_ exist in the first page of results).
-                if (settings && settings.projectName) {
-                    // Populate our project search with our project name.
-                    $projectFilterInput.val(settings.projectName);
-                }
-
                 fetchDataSourceContent(queryUri, authToken, "OctopusAllSpaces", null, null).done(function (data) {
                     const hasSpaces = !data.errorMessage;
                     if (hasSpaces) {
                         $spaceDropdown.prop("disabled", false);
                         appendSpaceData(data);
-
                         const selectedSpaceId = $spaceDropdown.val();
-                        fetchDataSourceContent(queryUri, authToken, "OctopusListProjectsInSpace", selectedSpaceId, $projectFilterInput.val()).done(appendProjectData);
+
+                        fetchDataSourceContent(queryUri, authToken, "OctopusAllProjectGroupsInSpace", selectedSpaceId, null).done(function (projectGroupData) {
```

Comment:
Similar to the pattern used for spaces, we firstly grab all project groups THEN load projects for the selected project group.